scripts/python_nbd_client.py

```diff
@@ -487,8 +488,9 @@ def connect(self, exportname):
         LOGGER.debug("NBD got size=%d transmission flags=%d",
                      self._size, self._transmission_flags)
         # ignore the zeroes
-        zeroes = self._recvall(124)
-        LOGGER.debug("NBD got zeroes: %s", zeroes)
+        if self._transmission_flags & NBD_FLAG_NO_ZEROES == 0:
```

Comment:
The bitwise operator `&` is used with a boolean value (`NBD_FLAG_NO_ZEROES == 0`), is this intended?

Reply:
That looks wrong to me.

Reply:
This would seem to be the point where a unit test would allow verification of the expected behaviour.

Reply:
You guys are right the intention is unclear, but the code is doing the correct thing.

Heres the doc for operator precedence https://docs.python.org/3/reference/expressions.html
" all comparison operations in Python have the same priority, which is lower than that of any arithmetic, shifting or bitwise operation"

There a numerous examples of similar expressions in this file on lines 207, 318, 654.

I will make the intention here clearer with some brackets.